index.html

```diff
@@ -96,11 +100,13 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
             <div><img src="images/tonic2.png" alt="tonic"></div>
             <div class="tonic-heading">Tonic</div>
```

Comment:
- [ ] As required by the previous reviewer, use a header tag instead of a div tag here e.g `<h2>`